lib/utils/fs_windows.go

```diff
@@ -39,3 +42,7 @@ func getPlatformLockFilePath(path string) string {
 	}
 	return path + lockPostfix
 }
+
+func getHardLinkCount(fi os.FileInfo) (bool, uint64) {
+	return false, 0
```

Comment:
Maybe add a comment

```
// hard links not supported for Windows
```

Reply:
Added a comment describing the current situation